### PR TITLE
User route changes

### PR DESCRIPTION
**Pull request by Martindevelops-songa**
password change for users done.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ app.use('/api/riders/auth', RiderAuthRoutes);
 app.use('/api/points', mpesaRoutes);
 app.use('/api/users/auth/OTP', UserOTPRoutes);
 app.use('/api/customer_agent', CustomerCareAgentRoutes);
-app.use('/api/rides', RiderRoutes);
+//app.use('/api/rides', RiderRoutes);
```
Comment (dennisngeno-songa) [inline src/app.ts:35]: Ride routes has been commented out

File: src/Users/controllers/UserPasswordControllers.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+const prisma = new PrismaClient();
+
+interface passwordChangeParams {
+  id: string;
+  oldPassword: string;
+  newPassword: string;
+  token?: string;
+}
+export const UserPasswordChange = async (req: Request, res: Response) => {
```
Comment (dennisngeno-songa) [inline src/Users/controllers/UserPasswordControllers.ts:15]: Should this endpoint be authorised?

**Review by dennisngeno-songa — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -18,8 +18,6 @@
   "license": "ISC",
   "dependencies": {
     "@prisma/client": "^4.14.0",
-    "@radarlabs/s2": "^0.0.5",
-    "@types/axios": "^0.14.0",
```
[the remark this answers is not in the document]
Comment (Martindevelops-songa) [reply package.json:22]: it's not working. It's causing the whole app to crash.

File: src/Users/controllers/UserPasswordControllers.ts (hunk 2 above)
Comment (Martindevelops-songa) [reply src/Users/controllers/UserPasswordControllers.ts:15]: yes. Will impliment that

File: src/Users/controllers/UserPasswordControllers.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    const isTokenValid = await VerifyToken(token);
+    
+    if (!isTokenValid) {
+      res.status(401).json({
+        message:
+          'Invalid session token. Please logout and login again then try changing your password',
+      });
+      return;
+    }
+
```
[the remark this answers is not in the document]
Comment (Martindevelops-songa) [reply src/Users/controllers/UserPasswordControllers.ts:115]: not yet done. we need to decide how users will reset passswords. we can email them or send a code

**Review by Martindevelops-songa — COMMENTED** [review]
reviewed

**Pull request merged**